modules/owner/profile.rb

```diff
@@ -1,79 +1,98 @@
-          case args[0].downcase
-          when 'nick', 'nickname'
-            begin
-              event.bot.profile.on(event.server).nickname = args.drop(1).join(' ')
-              event.respond("#{YuukiBot.config['emoji_tickbox']} Nickname changed to `#{args.drop(1).join(' ')}`")
-            rescue Discordrb::Errors::NoPermission
-              event.respond("#{YuukiBot.config['emoji_error']} I don't have permission to change my nickname!")
-            end
-          when 'avatar', 'avy'
-            url = args.drop(1).join(' ')
-            event.bot.profile.avatar = File.open(Helper.download_file(url, 'tmp'))
-          when 'game', 'playing'
-            event.bot.game = args.drop(1).join(' ')
-            event.respond("#{YuukiBot.config['emoji_tickbox']} Game set to `#{args.drop(1).join(' ')}`!")
-          when 'username', 'name'
-              username = args.drop(1).join(' ')
-            event.bot.profile.name = begin
-              username
-            rescue Error => e
-              event.respond("#{YuukiBot.config['emoji_error']} An error has occured!\n```ruby\n#{e}```")
-            end
-            event.respond("#{YuukiBot.config['emoji_tickbox']} Username _should_ be updated!")
-          when 'status', 'indicator', 'state'
-            state = args[1]
-            case state.downcase
-            when 'idle', 'away', 'afk' then event.bot.idle
-            when 'dnd'
-              state = 'Do Not Disturb'
-            when 'online' then event.bot.online
-            when 'invisible', 'offline' then event.bot.invisible
-            else
-              event.respond('Enter a valid argument!')
-              next
-            end
-            event.respond("#{YuukiBot.config['emoji_tickbox']} Status set to **#{state == 'Do Not Disturb' ? state : state.capitalize }**!")
+          # Let the default case handle as unknown.
+          args[0] = ''
+        end
+
+        error = YuukiBot.config['emoji_error']
+        tickbox = YuukiBot.config['emoji_tickbox']
+
+        case args[0].downcase
+        when 'nick', 'nickname'
+          begin
+            event.bot.profile.on(event.server).nickname = args.drop(1).join(' ')
+            event.respond("#{tickbox} Nickname changed to `#{args.drop(1).join(' ')}`")
+          rescue Discordrb::Errors::NoPermission
+            event.respond("#{error} I don't have permission to change my nickname!")
+          end
+        when 'avatar', 'avy'
+          url = args.drop(1).join(' ')
+          event.bot.profile.avatar = File.open(Helper.download_file(url, 'tmp'))
+        when 'game', 'playing'
+          event.bot.game = args.drop(1).join(' ')
+          event.respond("#{tickbox} Game set to `#{args.drop(1).join(' ')}`!")
+        when 'username', 'name'
+          username = args.drop(1).join(' ')
+          begin
+            event.bot.profile.name = username
+          rescue StandardError => e
+            event.respond("#{error} An error has occured!\n```ruby\n#{e}```")
```

Comment:
Didn't we make a standard method of displaying error embeds somewhere? Could be wrong.